### ISSUE TITLE
About Training SD

### ISSUE THREAD

**Issue by bigmover** [RedAIGC/Target-Driven-Distillation#9]
Hi guys! What are the points to pay attention to when training SD1.5? Can I just reuse the SDXL process?

**Comment by bigmover** [comment]
> > Hi guys! What are the points to pay attention to when training SD1.5? Can I just reuse the SDXL process?
> 
> Hello, we have not yet open-sourced the training code for SD1.5. However, you can modify it based on SDXL, as the basic training process is the same. Please note that you will need to adjust your batch size and learning rate accordingly.


Thank you for your kind reply!  Awesome work!
